examples/openfoam_supersonic_wedge.py

```diff
@@ -14,11 +14,17 @@
     Case, Dimension, FileName, FileClass
 )
 
-def main(case_dir='wedge'):
+def main(case_dir='wedge', test_example=False):
```

Comment:
I think it'd be better if this was a ``n_iter`` parameter rather than a flag. (See, for example, http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html)